src/state.ts

```diff
@@ -91,7 +91,7 @@ const state = new Proxy(overlay, {
     },
 })
 
-export async function getAsync(property) {
+export async function getAsync<K extends keyof State>(property: K): Promise<State[K]> {
     if (notBackground())
```

Comment:
If this is false, I don't think it returns a promise. We should probably wrap it in one for consistency? OTOH it's unlikely to ever be called from the background.

Reply:
It's an async function, so it can only return promises. Everything that isn't already a promise gets wrapped in one, while existing promises are returned as-is.

Reply:
Eugh, of course it is. I should not try to read code late at night. Thanks : )